spark-plugins/docs/NGramTransform-sparkcompute.md

```diff
@@ -0,0 +1,52 @@
+# NGramTransform Spark Compute
+
+Description
+-----------
+Transforms the input features into n-grams, where n-grams is a sequence of n tokens (typically words) for some integer 'n'.
+
+Use Case
+--------
+User wants to use tokens such as bigrams/ngrams instead of just unigrams while developing features for supervised Machine Learning models.
```

Comment:
@shashankmoghe please could you update this with the use case? the code looks fine, it is just missing the use case.